### DISCUSSION TITLE
Please tell me your favorite anime!!!

### DISCUSSION THREAD

**Discussion by 2220IKE** [NAIST-SE/AttractingContributors#562] [General]
Please comment in my repo.
https://github.com/2220IKE/AttractingContributors-anime/blob/main/README.md

**Comment by MikamiTetsuro365** [comment]
I love Cowboy Bebop!
![63da4d86da32dbe1d8968ca08bf85e49](https://user-images.githubusercontent.com/32039881/174527718-9646662a-f2be-47e9-8faa-89db117dec13.gif)

**Comment by zima168** [comment]
My favorite anime of this season is summer time rendering: https://summertime-anime.com

**Comment by yohanssen** [comment]
My favorite anime is 'attack on titan'
